### PR TITLE
Update README.md inside of src/test/

### PR DESCRIPTION
**Pull request by Christewart**
Improves clarity for README in the test directory

### CODE REVIEW

**Comment by sipa** [comment]
Can you stick to the text width of 80 characters, and provide a more accurate commit message? Also, it's not improving clarify - the text in README.md right now is simply wrong.

File: src/test/README.md [hunk 1]
```diff
@@ -5,18 +5,15 @@ sense to simply use this framework rather than require developers to
...
-examples of this pattern, examine uint160_tests.cpp and
-uint256_tests.cpp.
-
-Add the source files to /src/Makefile.test.include to add them to the build.
+test_bitcoin.cpp. To add a new unit test file to our test suite you need 
+to add the file to `src/Makefile.test.include`.  The pattern is to create 
+one test file for each class or source file for which you want to create 
+unit tests.  The file naming convention is `<source_filename>_tests.cpp` 
+and such files should wrap their tests in a test suite 
+called `<source_filename>_tests`. For an examples of this pattern,
```
Comment (paveljanik) [inline src/test/README.md]: an example

File: src/test/README.md [hunk 2]
```diff
@@ -5,18 +5,15 @@ sense to simply use this framework rather than require developers to
...
-directives).  The pattern is to create one test file for each class or
-source file for which you want to create unit tests.  The file naming
-convention is "<source_filename>_tests.cpp" and such files should wrap
-their tests in a test suite called "<source_filename>_tests".  For an
-examples of this pattern, examine uint160_tests.cpp and
-uint256_tests.cpp.
-
-Add the source files to /src/Makefile.test.include to add them to the build.
+test_bitcoin.cpp. To add a new unit test file to our test suite you need 
+to add the file to `src/Makefile.test.include`.  The pattern is to create
```
Comment (paveljanik) [inline src/test/README.md]: two spaces?

**Comment by paveljanik** [comment]
Please squash all commits after fixing nits.

**Comment by Christewart** [comment]
For some reason I couldn't squash to a more descriptive commit message, I tried following bitcoin core's guide and got an error saying I couldn't pick the specific commit I wanted to show

**Comment by MarcoFalke** [comment]
try `git commit --amend`

**Pull request merged**